catalog/admin/module/catalog-admin-module-sources/src/main/webapp/js/model/Service.js

```diff
@@ -87,47 +81,44 @@ define(function (require) {
                 return;
             }
             var configUrl = [model.configUrl, "createFactoryConfiguration", model.get("fpid")].join("/");
-            return $.ajax({type: 'GET',
+            return $.ajax({
+                type: 'GET',
                 url: configUrl
             });
         },
 
-        makeEnableCall: function(){
+        makeEnableCall: function () {
             var model = this;
             var pid = model.get('id');
             var url = [model.configUrl, "enableConfiguration", pid].join("/");
             if (pid) {
                 return $.ajax({
                     url: url,
                     dataType: 'json'
-                }).done(function(){
+                }).done(function () {
                     // massage some data to match the new backend pid.
                     model.trigger('enabled');
                     //enabling the model means the PID will be regenerated. This model no longer exists on the server.
                     model.destroy();
-                }).fail(function(){
```

Comment:
The promise is being returned to the caller so it's available for the caller to handle the fail case. It looks like the fail case was being tapped to log some error. Since it wasn't really doing anything and it doesn't prevent callers from handling the fail case i figure it's safe to remove.